lib/util/mail.php

```diff
@@ -104,6 +104,42 @@ function mail_utf8($to, $from_user, $from_email, $subject = '(No subject)', $mes
         return mail_ses($to, $subject, $message);
     }
 
+    if (getenv('MAIL_DRIVER') === 'mailcatcher') {
+        $transport = (new Swift_SmtpTransport('mailcatcher', 1025));
+        $mailer = new Swift_Mailer($transport);
+
+        $message = (new Swift_Message($subject))
+            ->setFrom([$from_email => $from_user])
+            ->setTo($to)
+            ->setBody($message);
+
+        return $mailer->send($message);
+    }
+
+    if (getenv('MAIL_DRIVER') === 'mailcatcher') {
```

Comment:
These repeated if blocks are intentional?

Reply:
Yes, SMTP Mailcatcher connection only if **mailcatcher** driver is in _.env_

Reply:
But why three times (107-117, 119-129, 131-141)? And if three times are necessary, why not use a loop?

Reply:
Sorry for three times repeated **if** blocks. First message from @meleu  was not understood correctly, because I don't see whole part of code in discussion. Already fixed, thank you!

Reply:
Sorry for three times repeated **if** blocks. First message from @meleu  was not understood correctly by me, because I don't see whole part of code in discussion.